tokio/src/runtime/basic_scheduler.rs

```diff
@@ -161,7 +161,7 @@ impl Spawner {
         F::Output: Send + 'static,
     {
         let (task, handle) = task::joinable(future);
-        self.scheduler.schedule(task);
+        self.scheduler.schedule(task, true);
```

Comment:
The meaning of the bool parameter here is a little unclear at the callsite, I'd prefer it if it was clearer what the "false" means. But, I don't think the other options I can think of (having separate `schedule` and `schedule_spawn` fns, or using an enum) are really worth it...